docs/FAQ.md

```diff
@@ -220,12 +220,12 @@ To resolve the error,
 
 Details of `--signature-verification` can be found by running `man dockerd`.
 
-## Delete of replication fails with Forbidden status , although the replication gets deleted. 
+## Delete of replication fails with Forbidden status , although the replication gets deleted using CLI or Remove-Azure. 
 
```

Comment:
I think you meant to use "Remove-AzureRmContainerRegistryReplication"

Reply:
Ifiexed it